src/processing/script/strict.js

```diff
@@ -0,0 +1,19 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+import { Syntax } from './tools/esotope';
+
+export const USE_STRICT_DIRECTIVE = 'use strict';
+
+export function hasStrictDirective (ast) {
```

Comment:
Do we really need separate file for this? Maybe move it to `index.js` and name it `isStrictMode`